sourced/ml/cmd_entries/repos2roles_and_ids.py

```diff
@@ -0,0 +1,29 @@
+import logging
+from uuid import uuid4
+
+from pyspark import Row
+
+from sourced.ml.extractors.roles_and_ids import RolesAndIdsExtractor
+from sourced.ml.transformers import Ignition, UastExtractor, UastDeserializer, \
+    HeadFiles, Uast2BagFeatures, Cacher, UastRow2Document, Mapper, CsvSaver
+from sourced.ml.utils import create_engine
+from sourced.ml.utils.engine import pause
+
+
+@pause
+def repos2roles_and_ids_entry(args):
+    log = logging.getLogger("repos2roles_and_ids")
+    engine = create_engine("repos2roles_and_ids-%s" % uuid4(), **args.__dict__)
+    extractors = [RolesAndIdsExtractor(args.split)]
+
+    ignition = Ignition(engine, explain=args.explain)
```

Comment:
Same here, `log` is not used, `engine` can be moved inside.

Reply:
We code everywhere as here:
https://github.com/src-d/ml/blob/master/sourced/ml/cmd_entries/repos2bow.py#L33
https://github.com/src-d/ml/blob/master/sourced/ml/cmd_entries/repos2coocc.py#L16
https://github.com/src-d/ml/blob/master/sourced/ml/cmd_entries/repos2df.py#L15

I think it is better to keep `engine` and `Ignition` separate.